editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "vision-zero-editor",
-  "version": "2.4.1",
+  "version": "2.4.3",
```

Comment:
i merged main into my branch out of habit, so all changes here and below are because of that